backend/Makefile

```diff
@@ -0,0 +1,16 @@
+localConfigName ?= local
+
+run:
+	CONFIG_NAME=$(localConfigName) docker-compose up --build -d
```

Comment:
Suggestion: Use docker compose instead of docker-compose

Reply:
Read about it, removed :)